test/helpers/points-helper.js

```diff
@@ -0,0 +1,19 @@
+// TODO: Finish filling in these methods
```

Comment:
Nah, TODO for this story, created it as a placeholder and as part of the tests that I am writing I will create and add new methods.

I add TODO's so I can do a search before a commit, just to ensure I don't miss anything :)